### DISCUSSION TITLE
Poursuite des évolutions de DMP OPIDoR : priorisation et enrichissement des « user stories »

### DISCUSSION THREAD

**Discussion by fcinist** [OPIDoR/DMPOPIDoR#206] [Annonces]
Depuis novembre 2021, une nouvelle version de [DMP OPIDoR](https://dmp.opidor.fr/) a été mise en ligne et propose, outre les modèles « classiques » de DMP, un modèle de DMP structuré et standardisé.

Cette version a pour objectif de rendre possible la production de DMP structuré représenté dans  un langage exploitable par les machines et permettant l’adaptation à des besoins communautaires disciplinaires. La prochaine étape pour optimiser cette version sera guidée par vos retours et besoins.

Nous avons déjà collecté un certain nombre de user stories issus des échanges et des retours de tests relatifs à cette nouvelle version. Vous pouvez les retrouver à la fin de cette annonce et nous vous invitons en tant qu’acteurs de la recherche (scientifiques, fournisseurs de services, personnels d’appui, financeurs, DPO …) :
- à sélectionner les 4 user stories qui vous semblent prioritaires (indiquez par leur lettre (A à I) les 4 user stories à privilégier), 
- à proposer de nouveaux scenarii non présents dans la liste mais qui vous semblent essentiels,
- ou tout simplement à participer à la discussion et aux échanges relatifs aux évolutions de DMP OPIDoR.

Nous espérons que vous serez nombreux à contribuer à ces échanges sur GitHub qui nous aideront à prioriser et planifier les futurs développements de DMP OPIDoR.

Toute l’équipe DMP OPIDoR vous remercie par avance.

**Liste des « user stories »**

> A. En tant qu’utilisateur, je souhaite que la réponse à une question puisse être commune à une partie ou à tous les produits de recherche pour cette question 

> B. En tant qu’utilisateur, je souhaite pouvoir interroger des référentiels (équipements techniques, vocabulaires, standards, entrepôts…) avec des critères de sélection et choisir le service/réponse qui me convient 

> C. En tant qu’utilisateur, je souhaite exporter mon PGD au format XML, CSV et dans un format ouvert (.odt)

> D. En tant qu’utilisateur, je souhaite pouvoir produire plusieurs versions du PGD, notamment les versions correspondant à des livrables pour les agences de financement, sans avoir à tout ressaisir (contrôle de versions des plans)

> E. En tant d'administrateur de catalogue/portail de données de recherche, je souhaite récupérer les informations concernant la description des produits de recherche pour référencer les données produites par mon établissement

> F. En tant qu’utilisateur, je souhaite ne pas avoir à saisir deux fois des informations que j’ai déjà saisies dans le formulaire de déclaration du traitement des données personnelles 

> G. En tant qu’utilisateur, je souhaite un meilleur affichage des recommandations, commentaires, formulaires et des actions « runs » (ex : notification, calcul de coût)

> H. En tant qu’administrateur pour un organisme, proposant un modèle de PGD pour mon  organisme, je souhaite avoir la fonctionnalité « question conditionnelle » pour simplifier la rédaction du PGD par le chercheur (Par exemple pour les données personnelles)

> I. En tant qu'utilisateur, je souhaite avoir la possibilité d’associer des images, des documents à un PGD (par exemple, une image du plan expérimental) et pouvoir les télécharger dans le PDF du PGD

**Comment by LaetitiaBracco** [comment]
B / D / G + ajouter des infobulles pour expliquer certains champs (par exemple pour de nombreux chercheurs, le "plan expérimental" n'est pas du tout parlant). Merci !

**Comment by equinton** [comment]
D, C, I, E
Pour E, j'aimerai pouvoir récupérer les plans de mon unité de recherche, et pas de l'établissement complet (dont je ne suis pas l'administrateur)
Et en subsidiaire, J : le PGD structure...

**Comment by clairekuchkly** [comment]
I,D et en plus le PGD de structure plus adapté au plateforme.

**Comment by CamilleTedesco** [comment]
B / D / G / H
Je rejoins l'idée de Laëtitia sur les infobulles, ce serait très intéressant pour les chercheurs.
Merci

**Reply by CamilleTedesco** [reply]
J'aimerai également avoir la possibilité de créer des dossiers/sous-dossiers pour organiser les PGD que je relis. Je commence à en avoir quelques-unes (plus d'une trentaine) et je m'y perds un peu sans une possibilité concrète d'organisation...

**Reply by CamilleTedesco** [reply]
Une autre demande est remontée : pouvoir avoir une troisième option de réponse dans la partie Rédiger du PGD
- réponses communes aux produits de recherche
- réponses individuelles pour chaque produit
- sélectionner les produits de recherches qui peuvent avoir des réponses communes

**Reply by CamilleTedesco** [reply]
Je vous transmets encore une autre demande : que tous les membres de DMP-Opidor puissent participer à l'élaboration du référentiel partenaires (à chaque fois qu'un nouveau partenaire est créé dans un PGD, cette création rejoint le référentiel commun)

**Comment by FGibier** [comment]
B, G, H
Merci !

**Comment by fehrenmann** [comment]
C, D, E, F et I. Je rejoins le commentaire d'Eric pour le E (pouvoir récupérer les plans de mon unité de recherche)
Merci à vous

**Comment by ConstanceCOURNEDE** [comment]
B, D, F, G
et les infobulles sont une bonne idée.

**Comment by amarois** [comment]
B, D, E, G
pour E : pouvoir récupérer les DMP de mon unités, donc
En fait, il faudrait avoir la possibilité d'être référent structure (cf. fonctionnement de HAL), ou au moins de demander à être notifié si un DMP est rattaché à une structure de recherche (ou, à défaut, si la structure est mentionnée; je pense à des alerte sur mot-clés). Il faudrait pouvoir revenir sur le DMP, ne voir que le titre peut-être, et pouvoir contacter le resp. du DMP à minima
et les infobulles

**Comment by froggypaule** [comment]
E, F, H, D : mes choix se justifient principalement par l'optimisation des données à renseigner : empêcher la duplication, favoriser la réutilisation de données déjà existantes

quelques user stories (déclaration : nous (IFB, Institut Français de Bioinformatique), avons démarré une conversation à ce sujet) ; mais elles pourraient interesser d'autres :  

- en tant qu'administrateur, je souhaite importer des trames de DMP, ou de parties de DMP, développées sur d'autre outils (en l'occurence ici, DSW)
- en tant qu'administrateur, je souhaite une certaine latitude (ie, dans un cadre à définir) à élaborer des formulaires spécialisés
- En tant d'administrateur ou utilisateur de DMPs, je souhaite récupérer les informations concernant la structure qui a généré et décrit les produits de recherche concernés par mon DMP (articulation PGD structure PGD projet)

**Comment by carenes** [comment]
D, E, G
pour E, je rejoins les remarques faites sur le fait de récupérer les plans, si le ou les auteurs nous y autorisent

**Comment by scocaud** [comment]
B, F, I, H

**Comment by equinton** [comment]
Bonjour,

Hors fil de discussion (quoique), nous avons besoin assez rapidement d'un modèle de schéma stratégique (appelé communément PGD structure) : ils sont la pierre angulaire des PGD, et ces derniers ont vocation à renvoyer vers les schémas stratégiques pour tout ce qui concerne la conservation des données, la sauvegarde, la sécurité, etc.

Dans mon institut (INRAE), je note une très forte effervescence sur ce sujet actuellement, et plus tôt nous aurons un canevas "structuré", mieux ce sera. Je suis prêt à travailler à ce sujet si besoin, mes fonctions m'imposant d'être moteur vis à vis des unités de mon département de recherche.

Cordialement,

Éric Quinton.

**Reply by JacquemotMC** [reply]
Bonjour,

Nous vous remercions pour votre participation à cette discussion. Nous souhaitons aborder la problématique des PGD structure ou schéma stratégique. Vous êtes plusieurs à avoir exprimer ce besoin. Si vous êtes d'accord, nous pourrions convenir d'un échange incluant les différentes personnes concernées. Ainsi il serait possible d'établir une trame commune. Nous allons vous envoyer un Evento.
Cordialement,
Christine Jacquemot

**Reply by equinton** [reply]
ok pour moi, pas de souci. Mon adresse pro est prenom.nom et mon institut.fr

**Reply by froggypaule** [reply]
Bonjour tout le monde. Nous aussi travaillons sur les PGD structure ; très interessée par cette discussion ! merci !

**Comment by CamilleTedesco** [comment]
Bonjour, 

Je vous remercie pour cette information et vous confirme que je suis tout à fait d'accord pour avoir un échange sur ces thématiques. 

Cordialement, 

7.7 - 03/02/2022 

Camille TEDESCO 

Bibliothèque Richter Droit Économie Gestion 

Services : Formation aux usagers, Appui à la recherche 

Tél : [ tel:+33 (0)4 11 75 99 78 | +33 (0)4 11 75 99 78 ] 

[ ***@***.*** | ***@***.*** ] 

[ http://www.umontpellier.fr/ ] 
	[ https://bibliotheques.edu.umontpellier.fr/ ] 
	

UNIVERSITÉ DE MONTPELLIER 

Bibliothèque Universitaire Richter 
60 rue des Etats Généraux 
34000 Montpellier 

[ https://bibliotheques.edu.umontpellier.fr/ | https://bibliotheques.edu.umontpellier.fr/ ] 

[ https://mail.umontpellier.fr/mail# ] [ https://mail.umontpellier.fr/mail# ] [ https://mail.umontpellier.fr/mail# ] [ https://mail.umontpellier.fr/mail# ] [ https://mail.umontpellier.fr/mail# ] [ https://orcid.org/0000-0001-9931-4111 ]

**Comment by clairekuchkly** [comment]
Bonjour,

Je suis également interessé.
Bien cordialement,

Claire Kuchly, IE
En télétravail majoritairement les lundi/mardi
Bioinformaticienne
GeT-PlaGe, US INRAE 1426
INRAE Auzeville
Chemin de Borde Rouge, Auzeville
31326 Castanet-Tolosan Cedex
Contact : ***@***.******@***.***>
Tel : 0561285591

[Logo-Signature-INRAE]    Institut National de Recherche pour l’Agriculture, l’Alimentation et l’Environnement

De : Eric Quinton ***@***.***>
Envoyé : mardi 15 février 2022 11:22
À : OPIDoR/DMPOPIDoR ***@***.***>
Cc : Claire Kuchly ***@***.***>; Comment ***@***.***>
Objet : Re: [OPIDoR/DMPOPIDoR] Poursuite des évolutions de DMP OPIDoR : priorisation et enrichissement des « user stories » (Discussion #206)


ok pour moi, pas de souci. Mon adresse pro est prenom.nom et mon institut.fr

—
Reply to this email directly, view it on GitHub<https://github.com/OPIDoR/DMPOPIDoR/discussions/206#discussioncomment-2179891>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXIBURW33PO6D3UBVO435LDU3ISMXANCNFSM5M5F72SQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by scocaud** [comment]
Bonjour,

Partante pour participer aussi bien entendu.

Sylvie Cocaud

[Logo-Signature-INRAE]

Sylvie COCAUD
Documentaliste
Pôle Numérique pour la science (Num4Sci)
DIrection pour la Science Ouverte (DipSO)

***@***.******@***.***>
https://orcid.org/0000-0002-7961-2821

INRAE Centre Grand Est-Nancy
54280 Champenoux
inrae.fr<https://www.inrae.fr/>



De : ckuchly ***@***.***
Envoyé : mardi 15 février 2022 11:29
À : OPIDoR/DMPOPIDoR ***@***.***>
Cc : Sylvie Cocaud ***@***.***>; Comment ***@***.***>
Objet : Re: [OPIDoR/DMPOPIDoR] Poursuite des évolutions de DMP OPIDoR : priorisation et enrichissement des « user stories » (Discussion #206)

Bonjour,

Je suis également interessé.
Bien cordialement,

Claire Kuchly, IE
En télétravail majoritairement les lundi/mardi
Bioinformaticienne
GeT-PlaGe, US INRAE 1426
INRAE Auzeville
Chemin de Borde Rouge, Auzeville
31326 Castanet-Tolosan Cedex
Contact : ***@***.******@***.***<mailto:***@***.******@***.***>>
Tel : 0561285591

[Logo-Signature-INRAE] Institut National de Recherche pour l’Agriculture, l’Alimentation et l’Environnement

De : Eric Quinton ***@***.***<mailto:***@***.***>>
Envoyé : mardi 15 février 2022 11:22
À : OPIDoR/DMPOPIDoR ***@***.***<mailto:***@***.***>>
Cc : Claire Kuchly ***@***.***<mailto:***@***.***>>; Comment ***@***.***<mailto:***@***.***>>
Objet : Re: [OPIDoR/DMPOPIDoR] Poursuite des évolutions de DMP OPIDoR : priorisation et enrichissement des « user stories » (Discussion #206)


ok pour moi, pas de souci. Mon adresse pro est prenom.nom et mon institut.fr

—
Reply to this email directly, view it on GitHub<https://github.com/OPIDoR/DMPOPIDoR/discussions/206#discussioncomment-2179891>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXIBURW33PO6D3UBVO435LDU3ISMXANCNFSM5M5F72SQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
You are receiving this because you commented.Message ID: ***@***.******@***.***<mailto:***@***.******@***.***>>>


—
Reply to this email directly, view it on GitHub<https://github.com/OPIDoR/DMPOPIDoR/discussions/206#discussioncomment-2179919>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAYID2XAGBFLBFHZDCNT7ELU3ITFLANCNFSM5M5F72SQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by carenes** [comment]
Je suis également intéressée pour participer aux échanges sur les DMP structure, merci pour cette proposition d'échange !

**Comment by Fsebire** [comment]
D, G, H, I 
En tant que « assistant conseil », lorsque j’ai déjà relu la V1 d’un PGD et que l’on me demande un an après de relire la V2, j’aimerais avoir un aperçu rapide de ce qui a été modifié depuis ma dernière relecture. Actuellement, je peux, en ouvrant chaque question, voir la dernière date de modification mais je ne peux pas filtrer les questions pour afficher uniquement celles modifiées récemment

**Comment by lmenard** [comment]
Bonjour et merci pour cette initiative. Dans le cadre du projet Européen H2020 e-shape (https://e-shape.eu/) dans le domaine de l’observation de Terre et de ses applications dans 7 thémes sociétaux nous avons demandés aux partenaires qui développent les applications pilotes (32 au total) de remplir chacun un DMP. Nous avons opté pour un format hybride basé à la fois sur FAIR Principles et sur GEO Data Management Principles. Nous y avons beaucoup appris et travaillons à concrétiser et partager cette connaissance, les concepts, les outils et les améliorations envisagées. Je n’ai pas de domaine particulier à sélectionner dans la liste OPIDoR ci dessus n’ayant pas utilisé l’outil mais je serai très intéréssé à suivre les discussions et à partager nos expériences. Cordialement.

**Comment by erlefloch** [comment]
A, B, D, F
A et B: articulation PGD structure(en particulier infrastructure)/ PGD de projet en évitant de réécrire , pour appeler des bouts de PGD de structure dans le PGD de projet
F: bloc correspondant à la partie gestion des données personnelles institutionnel et aussi réutilisable que possible
Être capable d'articuler des PGD DSW et OPIDoR nous intéresse aussi
DSW plutôt pour des projets, et OPIDoR plutôt pour des structures (?)

Par Anne-Françoise et Erwan

**Comment by amarois** [comment]
Je suis également intéressée pour participer aux échanges sur les DMP structure

**Comment by froggypaule** [comment]
Bonjour, je voulais signaler cette démarche vers un ma DMP : https://www.openaire.eu/blogs/argos-automates-the-writing-of-dmps-1

**Reply by abInist** [reply]
Bonjour Paulette, merci pour cette information. Nous connaissons ce projet et nous le suivons.

**Reply by froggypaule** [reply]
super :)

**Comment by sguyader** [comment]
Bonjour,

Pour les user stories : B-C-D-H.

Une requête : faire des mini-tutoriels pour aider à la rédaction de OGD de structure, avec des exemples pour différents types ou niveaux de structures (exemple : unité de recherche, pôle technique ou groupe rattaché à une unité), une FAQ, des exemples de ce qui est correct ou n'est pas correct pour certaines rubriques.

Partant pour participer.

Sébastien GUYADER
Chargé de Recherche / Référent données opérationnel
UR 1321 Agrosystèmes tropicaux
INRAE Centre Antilles-Guyane
F-97170 Petit-Bourg
Guadeloupe

**Comment by chamois31** [comment]
D / B / I / G
plus la possibilité de créer des groupes de Research output pour structurer le PGD dans l'interface (puis dans le doc)
